syntax = "proto2";

package msg.proto;

option optimize_for = CODE_SIZE;

message FileDownMsg{
    required string name = 1;
    required int64 startPos =2;
    optional int64 size = 3;
    optional bytes data = 4;
}
